android-core/src/main/java/com/mparticle/internal/KitFrameworkWrapper.java

```diff
@@ -646,6 +647,20 @@ public void reset() {
         }
     }
 
+    @Override
+    public void execute(String viewName,
+                        Map<String, String> attributes,
+                        Runnable onUnload,
+                        Runnable onLoad,
```

Comment:
Want to confirm on which thread these Runnable executes because all the callbacks needs to execute on MainThread as partner updates their UI on the response.